sdk/src/wallet/account/operations/balance.rs

```diff
@@ -225,6 +225,9 @@ impl Account {
         let mut locked_native_tokens = NativeTokensBuilder::new();
 
         for locked_output in &account_details.locked_outputs {
+            if account_balance.potentially_locked_outputs.contains_key(locked_output) {
```

Comment:
```suggestion
            // Skip potentially_locked_outputs because their amounts isn't added to the balance
            if account_balance.potentially_locked_outputs.contains_key(locked_output) {
```